AndroidBound/src/main/java/solutions/alterego/androidbound/android/ui/BindableTextView.java

```diff
@@ -3,140 +3,143 @@
 import android.content.Context;
 import android.content.res.ColorStateList;
 import android.graphics.Typeface;
+import android.graphics.drawable.Drawable;
+import android.graphics.drawable.StateListDrawable;
+import android.os.Build;
+import android.support.v7.widget.AppCompatTextView;
 import android.util.AttributeSet;
 import android.view.View;
 import android.widget.TextView;
 
 import rx.Observable;
 import rx.subjects.PublishSubject;
-import solutions.alterego.androidbound.interfaces.ICommand;
 import solutions.alterego.androidbound.binding.interfaces.INotifyPropertyChanged;
+import solutions.alterego.androidbound.interfaces.ICommand;
 
-public class BindableTextView extends TextView implements INotifyPropertyChanged, View.OnClickListener, View.OnLongClickListener {
-
-    private PublishSubject<String> propertyChanged = PublishSubject.create();
-
-    private boolean disposed;
-
-    private ICommand onClick = ICommand.empty;
+public class BindableTextView extends AppCompatTextView implements INotifyPropertyChanged {
 
-    private ICommand onLongClick = ICommand.empty;
+    private BindableViewDelegate mDelegate;
 
     public BindableTextView(Context context) {
-        super(context);
+        this(context, null);
     }
 
     public BindableTextView(Context context, AttributeSet attrs) {
         super(context, attrs);
+        mDelegate = createDelegate(this);
     }
 
     public BindableTextView(Context context, AttributeSet attrs, int defStyle) {
         super(context, attrs, defStyle);
+        mDelegate = createDelegate(this);
     }
 
-    public Typeface getTypeface() {
-        return super.getTypeface();
+    /****** beginning of the delegated methods, to be copy/pasted in every bindable view ******/
+
+    protected BindableViewDelegate createDelegate(View view) {
+        return new BindableViewDelegate(view);
```

Comment:
Do we really want to expose it this way? Maybe we could have a static factory method and hide the constructor. Is is something we want people to instantiate manually and or extend ?